src/AppInstallerRepositoryCore/Microsoft/Schema/Version.cpp

```diff
@@ -70,6 +72,18 @@ namespace AppInstaller::Repository::Microsoft::Schema
         THROW_HR(HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED));
     }
 
+    std::unique_ptr<IPortableIndex> Version::CreateIPortableIndex() const

```

Comment:
I would argue that maybe I made a mistake/was lazy here and that the "create interface" functions should probably live within the version erasing public interface (`SQLiteIndex` for example).

Reply:
Moved "create interface" functions to version erasing public interface